code/datums/components/aiming.dm

```diff
@@ -38,8 +38,8 @@
 	user.visible_message("<span class='warning'>[user] points [parent] at [target]!</span>")
 	to_chat(target, "<span class='userdanger'>[user] is pointing [parent] at you! If you equip or drop anything they will be notified! \n<b>You can use *surrender to give yourself up</b>.</span>")
 	to_chat(user, "<span class='notice'>You're now aiming at [target]. If they attempt to equip anything you'll be notified by a loud sound.</span>")
-	user.balloon_alert_to_viewers("[user] points [parent] at [target]!", ignored_mobs = list(user, target))
-	user.balloon_alert(target, "[user] points [parent] at you!")
+	user.balloon_alert_to_viewers("[user] points a gun at [target]!", ignored_mobs = list(user, target))
+	user.balloon_alert(target, "[user] points a gun at you!")
```

Comment:
I think this is better as [parent] in cases where the gun is something weird, like a magic wand

Reply:
Alright, reverted it, although I do think it feels a bit clunky in cases where there's wielded weapons with long names.